### DISCUSSION TITLE
v1.8.11 - Sequoia Compatibility & Bug Fixes

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#953] [General]
* Lowered CPU Usage
* Now when Extra Effect crashes it just gets disabled instead of restarting the app
* Fixed Microphone being used by Web Browers when eqMac is enabled
* Fixed AirPods going into switching loop for some users
* Fixed Devices with >2 channels having distorted audio
* Fixed issue that prevented eqMac from working on macOS Sequoia
* Fixed Expert Preset Import not working
* Fixed QuickLook sound not being routed when App Mixer is enabled
* Fixed QuickTime Player audio recording not working
* Added ability to choose how to route System Alert sounds
* Fixed Extra Effects state not being saved in some cases
* Added ability to adjust Super Presets while disabled to avoid constant switching
* Other minor bug fixes and improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/'>v1.8.11 - Sequoia Compatibility & Bug Fixes</a>.</em>

**Comment by pedrowrt** [comment]
Hi, im still having issues with the driver installation loop, tried restarting and manual installing in the library folders. The last version that the driver works is 1.8.8 and Driver 2.5.4.
I'm running Sonoma on iMac mid 2019 27"

**Comment by Sarkuaza** [comment]
I'm also experiencing a driver installation loop in 1.8.11

**Comment by Anthrobug** [comment]
Me too... endless beach balling after install.
Right after the first install, the UI comes up - looks a little blurry? - then it freezes, both the app and the menu item. Restart, no UI opens - app icon just shows dot under it but no main window & menubar icon shows beachball when hovering over it. 
M4 Mac mini, 16gb/256gb

**Reply by hehehendrik** [reply]
same here

**Comment by omscg** [comment]
Anyone find a fix for this? Getting the same on an M2 Mac Mini. I had to downgrade back to 1.8.8 to get it working.
